lib/Core/Searcher.h

```diff
@@ -223,14 +223,14 @@ namespace klee {
 
     private:
 
-    Searcher *baseSearcher;
+    std::unique_ptr<Searcher> baseSearcher;
```

Comment:
This is unrelated to this line: Can you move the implementation part of this searcher to the `.cpp` file.